docs/understand_mmcv/io.md

```diff
@@ -29,6 +35,20 @@ with open('test.yaml', 'w') as f:
     data = mmcv.dump(data, f, file_format='yaml')
 ```
 
+#### Load from other backends or dump to other backends
```

Comment:
We should contain a further study where we tell the uses how to use the ceph client to put checkpoint file (if only a file path is given).

Reply:
great suggestion

Reply:
Just to make it more clear. Users may have a locally saved checkpoint in `local-path`, and they want to put it to a ceph path `ceph-path`.

Reply:
got it